.eslintrc.json

```diff
@@ -1,19 +1,13 @@
 {
-  "plugins": ["node", "prettier"],
+  "plugins": ["prettier"],
   "parserOptions": {
     "sourceType": "module"
   },
   "env": {
     "node": true,
     "es6": true
   },
-  "extends": [
-    "plugin:import/errors",
-    "plugin:import/warnings",
-    "eslint:recommended",
-    "plugin:node/recommended",
-    "prettier"
-  ],
+  "extends": ["eslint:recommended", "standard", "prettier", "prettier/standard"],
   "rules": {
     "no-unused-vars": [
```

Comment:
The `no-unused-vars` can be removed because it is already part of `eslint-config-standard` with the same options: https://github.com/standard/eslint-config-standard/blob/3f4a3fed2c5d5fdf03e5c32e6c87d2fdc3ea4282/eslintrc.json#L158

Reply:
Yes, `any` always leads to false positives